Corona-Warn-App/src/test/java/de/rki/coronawarnapp/appconfig/internal/ConfigDataContainerTest.kt

```diff
@@ -17,96 +17,96 @@ class ConfigDataContainerTest : BaseTest() {
         )
         config.isValid(Instant.EPOCH) shouldBe false
-        config.isValid(Instant.EPOCH.plus(Duration.standardHours(1))) shouldBe false
-        config.isValid(Instant.EPOCH.plus(Duration.standardHours(24))) shouldBe false
-        config.isValid(Instant.EPOCH.plus(Duration.standardDays(14))) shouldBe false
+        config.isValid(Instant.EPOCH.plus(Duration.ofHours(1))) shouldBe false
+        config.isValid(Instant.EPOCH.plus(Duration.ofHours(24))) shouldBe false
+        config.isValid(Instant.EPOCH.plus(Duration.ofDays(14))) shouldBe false
 
-        config.isValid(Instant.EPOCH.minus(Duration.standardHours(1))) shouldBe false
-        config.isValid(Instant.EPOCH.minus(Duration.standardHours(24))) shouldBe false
-        config.isValid(Instant.EPOCH.minus(Duration.standardDays(14))) shouldBe false
+        config.isValid(Instant.EPOCH.minus(Duration.ofHours(1))) shouldBe false
+        config.isValid(Instant.EPOCH.minus(Duration.ofHours(24))) shouldBe false
+        config.isValid(Instant.EPOCH.minus(Duration.ofDays(14))) shouldBe false
     }
 
     @Test
     fun `updated at is based on servertime and offset`() {
         val config = ConfigDataContainer(
             serverTime = Instant.EPOCH,
-            localOffset = Duration.standardHours(1),
+            localOffset = Duration.ofHours(1),
             mappedConfig = mockk(),
             configType = ConfigData.Type.LAST_RETRIEVED,
             identifier = "localetag",
-            cacheValidity = Duration.standardSeconds(0)
+            cacheValidity = Duration.ofSeconds(0)
         )
-        config.updatedAt shouldBe Instant.EPOCH.plus(Duration.standardHours(1))
+        config.updatedAt shouldBe Instant.EPOCH.plus(Duration.ofHours(1))
     }
 
     @Test
     fun `device time correctness is checked via localOffset`() {
         val forOffset: (Duration) -> ConfigData = {
             ConfigDataContainer(
-                serverTime = Instant.EPOCH.plus(Duration.standardDays(1)),
+                serverTime = Instant.EPOCH.plus(Duration.ofDays(1)),
                 localOffset = it,
                 mappedConfig = mockk<ConfigMapping>().apply {
                     every { isDeviceTimeCheckEnabled } returns true
                 },
                 configType = ConfigData.Type.LAST_RETRIEVED,
                 identifier = "localetag",
-                cacheValidity = Duration.standardSeconds(0)
+                cacheValidity = Duration.ofSeconds(0)
             )
         }
 
-        forOffset(Duration.standardHours(1)).isDeviceTimeCorrect shouldBe true
+        forOffset(Duration.ofHours(1)).isDeviceTimeCorrect shouldBe true
 
         forOffset(Duration.ZERO).isDeviceTimeCorrect shouldBe true
 
-        forOffset(Duration.standardHours(2).minus(1)).isDeviceTimeCorrect shouldBe true
-        forOffset(Duration.standardHours(-2).plus(1)).isDeviceTimeCorrect shouldBe true
+        forOffset(Duration.ofHours(1)).isDeviceTimeCorrect shouldBe true
```

Comment:
For what reason has the setup changed here?

Reply:
It was done as part of some bulk find & replace update and it worked. But I updated the test now to have the same values as before